apps/api/src/main/resources/data.sql

```diff
@@ -546,6 +546,7 @@ FROM ((
              , 'ac86995f-60ae-4520-a434-c8abc98980b9' AS user_id
              , 'c0a80121-7e81-1c3e-817e-82cc11180004' AS billing_info_contact_info_id
              , 'c0a80121-7e81-1c3e-817e-82cc11180005' AS contact_info_contact_info_id
+             , false                                  AS paymentReminderSent
```

Comment:
This should probably follow the naming convention for the database: payment_reminder_sent

Tho, if calli approves of it as it is, it's fine. But check with her.

Reply:
It works either way, but might as well make it match the rest of the data
If it doesn't work after changing it to `payment_reminder_sent` then leave it as is @Sintry1